Package.swift

```diff
@@ -29,7 +29,7 @@ let package = Package(
         ),
         .binaryTarget(
             name: "AmazonChimeSDKMedia",
-            url: "\(hostingUrl)/media-without-bitcode/\(SDKMediaVersion)/spm/AmazonChimeSDKMedia-\(SDKMediaVersion).zip",
```

Comment:
is removing this intentional?

Reply:
Close this PR as we decided to open another github repo dedicated for no-video-codecs spm distribution